packages/katello/katello/katello-remove

```diff
@@ -120,38 +126,41 @@ CERT_FILES = [
 CONTENT = [
   '/opt/puppetlabs',
   '/opt/theforeman',
-  '/root/.cache/apipie_bindings',
   '/root/.hammer',
   '/usr/lib/pulp',
   '/usr/libexec/foreman-proxy',
+  '/usr/libexec/qpid',
   '/usr/share/candlepin',
-  '/usr/share/foreman-installer-katello',
+  '/usr/share/foreman',
   '/usr/share/foreman-installer',
   '/usr/share/foreman-proxy',
-  '/usr/share/foreman',
-  '/usr/share/katello-installer-base',
   '/usr/share/katello',
   '/usr/share/pulp',
   '/usr/share/puppet',
-  '/usr/share/qpid-tools',
   '/usr/share/qpid',
+  '/usr/share/qpid-tools',
   '/usr/share/tomcat',
-  '/var/cache/',
+  '/var/cache/candlepin',
+  '/var/cache/httpd',
+  '/var/cache/tomcat',
   '/var/lib/candlepin',
-  '/var/lib/foreman-maintain',
   '/var/lib/foreman',
+  '/var/lib/foreman-maintain',
+  '/var/lib/foreman-proxy',
   '/var/lib/katello',
   '/var/lib/puppet',
   '/var/lib/qpidd',
   '/var/lib/tomcat',
   '/var/opt/theforeman',
+  '/var/spool/squid',
   '/var/tmp/foreman-proxy'
 ]
 
 MOUNTS = [
```

Comment:
Why do we even track mounts? Don't you just want to check if a `content` is a mount, then remove `#{content}/*`